src/features/ProtectTransaction/utils.ts

```diff
@@ -19,122 +24,123 @@ import {
 import { ProtectTxError, NansenReportType } from './types';
 import { MALICIOUS_LABELS, WHITELISTED_LABELS } from './constants';
 
-export abstract class ProtectTxUtils {
-  public static getProtectTransactionFee(
-    sendAssetsValues: IFormikFields,
-    rate: number | undefined
-  ): { amount: BigNumber | null; fee: BigNumber | null } {
-    if (sendAssetsValues.amount === null || !isNumber(rate)) return { amount: null, fee: null };
-
-    const {
-      amount,
-      gasLimitField,
-      gasEstimates: { safeLow }
-    } = sendAssetsValues;
-
-    const gasPrice = sendAssetsValues.advancedTransaction
-      ? sendAssetsValues.gasPriceField
-      : sendAssetsValues.gasPriceSlider;
-
-    const fixedHalfDollar = PROTECTED_TX_FIXED_FEE_AMOUNT / rate;
-
-    let fixedFee = 0;
-    try {
-      fixedFee = parseFloat((parseFloat(amount) * PROTECTED_TX_FEE_PERCENTAGE).toString());
-    } catch (e) {
-      console.error(e);
-    }
-
-    const mainTransactionWei = parseFloat(
-      fromWei(Wei(totalTxFeeToWei(gasPrice.toString(), gasLimitField)), 'ether')
-    );
-
-    const protectedTransactionWeiMin = Math.min(safeLow, 10);
-    const protectedTransactionWei = parseFloat(
-      fromWei(Wei(totalTxFeeToWei(protectedTransactionWeiMin.toString(), gasLimitField)), 'ether')
-    );
-
-    return {
-      amount: bigify((fixedHalfDollar + fixedFee - mainTransactionWei).toString()),
-      fee: bigify(protectedTransactionWei.toString())
-    };
+export const getProtectTransactionFee = (
```

Comment:
Since we're changing this lets stay consistent with naming and use `getProtectTxFee` (or `getPTxFee`?)